components/tenant-fetcher/internal/tenant/service.go

```diff
@@ -62,10 +62,14 @@ func (s *service) Create(writer http.ResponseWriter, request *http.Request) {
 		http.Error(writer, fmt.Sprintf("Property %q not found in body or it is not of String type", s.config.TenantProviderTenantIdProperty), http.StatusInternalServerError)
 		return
 	}
+	customerId := gjson.GetBytes(body, s.config.TenantProviderCustomerIdProperty)
```

Comment:
Customer ID is mandatory and is always present - it is better to return error in case it is missing.